smoothscroll.js

```diff
@@ -78,8 +83,16 @@ document.addEventListener("DOMContentLoaded", function () {
     }
 });
 
+if (typeof exports !== "undefined") {
+  if (typeof module !== "undefined" && module.exports) {
+    exports = module.exports = smoothScroll;
+  }
+  exports.smoothScroll = smoothScroll;
+} else {
+  root.smoothScroll = smoothScroll;
+}
+
 // return smoothscroll API
 return smoothScroll;
 
-})();
-
+})(this);
```

Comment:
I'm probably missing something here, but what are you calling the function with `this`?